apps/studio/components/interfaces/QueryPerformance/QueryPerformanceFilterBar.tsx

```diff
@@ -98,6 +101,12 @@ export const QueryPerformanceFilterBar = ({
             onSaveFilters={onFilterRolesChange}
             className="w-56"
           />
+
+          {sort && (
+            <Button size="tiny" type="link" icon={<X size={12} />} onClick={clearSort}>
+              Clear sort
+            </Button>
+          )}
```

Comment:
I don’t see any visual feedback on _how_ the table is currently sorted. Should that be here?